docs/docs-beta/docs/integrations/libraries/fivetran.md

```diff
@@ -18,19 +18,65 @@ tags: [dagster-supported, etl]
 ---
 
 
+# Using Dagster with Fivetran
 
-The Dagster-Fivetran integration enables you to orchestrate data ingestion as part of a larger pipeline. Programmatically interact with the Fivetran REST API to initiate syncs and monitor their progress.
+This guide provides instructions for using Dagster with Fivetran using the `dagster-fivetran` library. Your Fivetran connector tables can be represented as assets in the Dagster asset graph, allowing you to track lineage and dependencies between Fivetran assets and data assets you are already modeling in Dagster. You can also use Dagster to orchestrate Fivetran connectors, allowing you to trigger syncs for these on a cadence or based on upstream data changes.
 
-### Installation
+## What you'll learn
 
-```bash
-pip install dagster-fivetran
-```
+- How to represent Fivetran assets in the Dagster asset graph, including lineage to other Dagster assets.
+- How to customize asset definition metadata for these Fivetran assets.
+- How to materialize Fivetran connector tables from Dagster.
+- How to customize how Fivetran connector tables are materialized.
 
-### Example
+<details>
+  <summary>Prerequisites</summary>
 
-<CodeExample filePath="integrations/fivetran.py" language="python" />
+- The `dagster` and `dagster-fivetran` libraries installed in your environment
+- Familiarity with asset definitions and the Dagster asset graph
+- Familiarity with Dagster resources
+- Familiarity with Fivetran concepts, like connectors and connector tables
+- A Fivetran workspace
+- A Fivetran API key and API secret. For more information, see [Getting Started](https://fivetran.com/docs/rest-api/getting-started) in the Fivetran REST API documentation.
```

Comment:
This is showing _undefined_
<img width="351" alt="image" src="https://github.com/user-attachments/assets/46fd9e79-a1f1-4fbf-8362-b7b8da9c399a" />